lv_gif.c

```diff
@@ -151,3 +154,16 @@ static lv_res_t lv_gif_signal(lv_obj_t * img, lv_signal_t sign, void * param)
 
     return LV_RES_OK;
 }
+
+/**
+ * Event function of the image
+ * @param img pointer to a image object
+ * @param event the triggering event
+ */
+static void lv_gif_event(lv_obj_t * img, lv_event_t event)
+{
+    if (event == LV_EVENT_LEAVE) {
```

Comment:
Please add this into `lv_gif_signal` as `LV_SIGNAL_LEAVE`.
If we use an event here the user can't set his own even callback.

Reply:
@kisvegabor Thanks for your feedback, I updated the LV_EVENT_LEAVE to LV_SIGNAL_LEAVE, please check.